scripts/portal.json

```diff
@@ -161,29 +161,507 @@
                     }
                 ]
             },
-            "url": "http://esapubs.org/archive/ecol/E090/118/Portal_rodents_19772002.csv"
+            "url": "https://raw.githubusercontent.com/weecology/PortalData/v1.0.1/Rodents/Portal_rodent.csv"
         },
         {
             "dialect": {},
-            "name": "species",
-            "schema": {},
-            "url": "https://ndownloader.figshare.com/files/3299483"
+            "name": "rodent_species",
+            "schema": {
+                "fields":[
+                    {
+                        "name":"species_code",
+                        "type":"char",
+                        "size":"2"
+                    },
+                    {
+                        "name":"scientific_name",
+                        "type":"char"
+                    },
+                    {
+                        "name":"taxa",
+                        "type":"char"
+                    },
+                    {
+                        "name":"common_name",
+                        "type":"char"
+                    },
+                    {
+                        "name":"census_target",
+                        "type":"int"
+                    },
+                    {
+                        "name":"unidentified",
+                        "type":"int"
+                    },
+                    {
+                        "name":"rodent",
+                        "type":"int"
+                    },
+                    {
+                        "name":"granivore",
+                        "type":"int"
+                    }
+                ]
+            },
+            "url": "https://raw.githubusercontent.com/weecology/PortalData/blob/v1.0.1/Rodents/Portal_rodent_species.csv"
```

Comment:
At least this link is wrong and causing the script to error. Can you update it to:

https://raw.githubusercontent.com/weecology/PortalData/v1.0.1/Rodents/Portal_rodent_species.csv

and check the others and make sure everything is running and ping me to check things again. Thanks.